### ISSUE TITLE
Amstatus doesn't show ip&port for all the players

### ISSUE THREAD

**Issue by Morabis** [Razish/japp#307]
It's a constant issue on our server that amstatus doesn't show the ip for every player.

http://puu.sh/gnfvi/e6659d7601.jpg

On the picture above the ip and port was shown where I masked with white rectangles but haven't shown for other players as you see.

**Comment by ensiform** [comment]
Is the server running openjk ?

If not, then that's why.

**Comment by EpicLoyd** [comment]
why?

**Comment by ensiform** [comment]
jamp/jampded doesn't retain the IP field reliably for mods.

**Comment by Morabis** [comment]
Yeah server is running 64 bit Openjk (date 8th of February) and Ja++ 64bit as well (date 20th of February).

**Comment by ensiform** [comment]
Could be related to your settings of not using userinfo validation possibly.

**Comment by Morabis** [comment]
It's value is 805306367 the default.

**Comment by EpicLoyd** [comment]
debug it
